### ISSUE TITLE
MD5 hash error

### ISSUE THREAD

**Issue by Barbarosse** [pmeier/pystiche#222]
Hi, I get this error when running the script given in the example for beginner.

`FileExistsError: bird1.jpg with a different MD5 hash already exists in /root/.cache/pystiche. If you want to overwrite it, set overwrite=True.`

**Comment by pmeier** [comment]
Hi, 

in #211 I've changed the download links to the original images to better reflect their origin. Apparently this also changed their resolution which then in turn results in a different MD5 hash. 

---

To overcome this you have to options:

1. You can set `overwrite=True` as the error message suggests:

    ```python
     images = demo_images()
     images.download(overwrite=True)
    ```
     With this image will simply overwritten ignoring the MD5.

2. You can delete the file `bird1.jpg` from the directory `/root/.cache/pystiche`. If you choose this option also delete `bird2.jpg` since its download link also changed.

**Comment by Barbarosse** [comment]
Thanks Pmeier ! I tried both suggestions both don't resolve the problem. 
Something strange occur when running the script ; two other images (content and style) I used in a previous Pytorch code reappear every time I run the code in the Pystiche cache folder.

**Comment by pmeier** [comment]
@Barbarosse This should be fixed with #228. If the problem still persists, feel free to re-open this.
